guides/common/modules/proc_configuring-installation.adoc

```diff
@@ -51,6 +57,43 @@ endif::[]
 +
 The script displays its progress and writes logs to `{installer-log-file}`.
 
+ifdef::katello,satellite,orcharhino[]
```

Comment:
There's an ordering issue with how it's written now that can lead users to still have the local database installed.

First we tell users to run the installer which will install a local PostgreSQL. Then we tell them to point their installation to a remote DB server. Now you still have an unused local PostgreSQL running.

Perhaps we should tell users to append the options mentioned below to the above command, similar to the SSL options. Another option is to make the command below complete (add `--foreman-initial-*` options to it) and run it **instead**.

I'll leave it up to writers to decide what's best for the reader.